### PR TITLE
Add API-endpoints for user-facing editable text

### PR DESCRIPTION
**Pull request by LarsSelbekk**
- refactor(bl-endpiont): delete redundant files
- feat(bl-endpoint): add PUT endpoint
- refactor(mongoose): use built-in timestamping
- feat(editable-text): add endpoints for editing user-facing text
- chore: run prettier

### CODE REVIEW

**Comment by LarsSelbekk** [comment]
[Legge til admin-bestemt banner på forsiden](https://trello.com/c/Nd6SQ13z/719-legge-til-admin-bestemt-banner-p%C3%A5-forsiden)

File: src/collections/editable-text/hooks/editable-text.put.hook.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import { BlDocumentStorage } from "../../../storage/blDocumentStorage";
+import { BlCollectionName } from "../../bl-collection";
+import { editableTextSchema } from "../editable-text.schema";
+
+export class EditableTextPutHook extends Hook {
+  private readonly editableTextStorage: BlDocumentStorage<EditableText>;
+
+  constructor(editableTextStorage?: BlDocumentStorage<EditableText>) {
+    super();
+    this.editableTextStorage = editableTextStorage
```
Comment (AdrianAndersen) [inline src/collections/editable-text/hooks/editable-text.put.hook.ts:13]: ??=

File: src/storage/mongoDb/mongoDb.blStorageHandler.ts [hunk 2]
```diff
@@ -212,6 +208,22 @@ export class MongoDbBlStorageHandler<T extends BlDocument>
     throw new BlError("not implemented");
   }
 
+  public async upsert(id: string, data: unknown): Promise<void> {
+    const result = await this.mongooseModel.replaceOne({ _id: id }, data, {
+      upsert: true,
+    });
+    console.log(result);
```
Comment (AdrianAndersen) [inline src/storage/mongoDb/mongoDb.blStorageHandler.ts:215]: 💩

**Review by AdrianAndersen — CHANGES_REQUESTED** [review]
Looks good! Just a few nits 🌞

**Comment by LarsSelbekk** [comment]
Updated the upsert method in mongoDb.blStorageHandler to be called put, and changed its error handling slightly because of the Mongo upgrade.

**Review by AdrianAndersen — APPROVED** [review]
Gucci rockenrolla 🎸

**Pull request merged**